reports/dataentry-statistics.json

```diff
@@ -394,6 +394,7 @@
    {"joinType":"INNER JOIN","schema":"amrs","alias":"t2","joinExpression":"t2.patient_id = t1.person_id",
```

Comment:
It is already defined in the patientList-encounter data set which we are joining against.